subconscious/model.py

```diff
@@ -287,13 +288,24 @@ def get_index_key(cls, column_name):
 
     @classmethod
     async def _get_ids_filter_by(cls, db, **kwargs):
+        order_by = kwargs.pop('order_by', None)
```

Comment:
Why do this via `kwargs`? Better to be explicit and put `order_by=None` in the method signature.